### DISCUSSION TITLE
4.2.9 upgrade problems

### DISCUSSION THREAD

**Discussion by StrangeMinds1960** [mastodon/mastodon#30488] [Q&A] [answered]
A while back I upgraded to the latest Ubuntu version (24.04?) and now get the following error during install -

mastodon@strangeminds:~/live$ yarn install --frozen-lockfile
Usage: yarn [options]

yarn: error: no such option: --frozen-lockfile

**Comment by mjankowski** [comment]
Did you sort this out?

I forget the exact version history/order, but at some point in time `yarn` version was upgraded and the command changed. That said, the change should have been in upgrade notes...

**Comment by StrangeMinds1960** [comment, marked as answer]
I think I was talked through upgrading Yarn, either way, it was fixed and I'm currently on 4.3.1
